docs/sql/statements/copy-from.rst

```diff
@@ -515,6 +533,28 @@ This option specifies the format of the input file. Available formats are
 guessed from the file extension, the file will be processed as JSON.
 
 
+.. _sql-copy-from-header:
+
+``header``
+''''''''''
+
+This option can be used when the input file is formatted as csv. If set to
+``false`` the ``header`` option enables the parsing of csv files that have
+no header. The default value is ``true`` meaning that the input csv
+file's first row is assumed to be a csv header. If header option is ``true``
+and the ``column_ident`` is a superset of the csv header null values will be
+added, whereas if ``column_ident`` is a subset of csv header values will be
+ignored. In the case where header option is ``false`` and ``column_ident``
+is a superset of csv header, the operation is not allowed.
+
+An example of using input file with no header
```

Comment:
```suggestion
Used to indicate if the first line of a CSV file contains a header with the
column names. Defaults to ``true``.

If set to ``false``, the CSV must not contain column names in the first line
and instead the columns declared in the statement are used. If no columns are
declared in the statement, it will default to all columns present in the table
in their ``CREATE TABLE`` declaration order.

If set to ``true`` the first line in the CSV file must contain the column
names. You can use the optional column declaration in addition to import only a
subset of the data.

If the statement contains no column declarations, all fields in the CSV are
read and if it contains fields where there is no matching column in the table,
the behavior depends on the ``column_policy`` table setting. If ``dynamic`` it
implicitly adds new columns, if ``strict`` the operation will fail.

An example of using input file with no header
```